docs/src/routes/docs/components/dropdown-shell.svx

```diff
@@ -0,0 +1,108 @@
+---
+name: DropdownShell
+relatedComponents:
+  - name: Dropdown
+    link: ./docs/components/dropdown
+---
+
+<script>
+  import { DropdownShell, Dropdown, Button } from 'attractions';
+  import { ChevronDownIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import ShadowPreview from 'src/components/docs/shadow-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+</script>
+
+# DropdownShell
+
+The container for the [`Dropdown`](./docs/components/dropdown) activator and content.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <DropdownShell let:toggle>
+      <Button on:click={toggle}>
+        this is the dropdown shell
+        <ChevronDownIcon size="24" class="ml dropdown-chevron" />
+      </Button>
+      <Dropdown right>
+        <div class="padded">
+          and this is the dropdown
+        </div>
+      </Dropdown>
+    </DropdownShell>
+    <DropdownShell>
+      Useless without a <code>&lt;Dropdown /&gt;</code>
+    </DropdownShell>
+  </div>
+  <div slot="source">
+
+```svelte
+<DropdownShell let:toggle>
+  <Button on:click={toggle}>
+    this is the dropdown shell
```

Comment:
Which brings me over here, actually. If we say here "this is the dropdown handle" instead of the shell (because it isn't the shell, the shell is around them both), we can get away with using the "handle" term in the previous comment